src/Components/DrillPageMinimal.js

```diff
@@ -41,6 +41,7 @@ export const DrillPageMinimal = props => {
     );
     navigation.setOptions({
       headerTitleAlign: 'center',
+      headerBackTitleVisible: false, // hide the "Back" text on iOS
```

Comment:
I think I had it on one or two more pages

I guess we should also hide it on every page where the title is dynamic (drill/training/program title ?)

Reply:
Or maybe everywhere to write it just once on the Navigation component

Reply:
Done everywhere. We can tweak it on V1.